patterns/1-initial/transitioning-contractor-code-to-innersource-model.md

```diff
@@ -0,0 +1,77 @@
+## Title
+
+Transitioning Contractor Code to InnerSource Model
+
+## Patlet
+
+Contract developers are often not motivated to engage in InnerSource activities, which may be beyond the scope of their contract. This patlet describes how you can focus on transitioning the contractor project after the fact to an InnerSource project.
```

Comment:
The idea of the Patlet is that the 1st sentence describes the problem, which the 2nd sentence describes the solution. This allows readers to quickly scan for problems they have and solutions they need.

Therefore I would suggest to rewrite the 2nd sentence as a summary of the solution outlined in this pattern.